.gitlab-ci.yml

```diff
@@ -142,7 +142,7 @@ linux-snap:
     - rust-stable
   artifacts:
     paths:
-    - scripts/parity_*_amd64.snap
+    - parity.zip
```

Comment:
Should we add something that indicates that it's a snap? (to don't confuse it with packed binary or sth)

Reply:
it only in artifacts on gitlab. for tests, we are not push in release